delphi-frontend/src/main/java/au/com/integradev/delphi/symbol/SymbolTableBuilder.java

```diff
@@ -185,7 +185,7 @@ private static boolean isPasFile(Path path) {
   }
 
   private void createUnitData(Path unitPath, boolean isSourceFile) {
-    if (unitPaths.add(unitPath) || isSourceFile) {
+    if (unitPaths.add(unitPath)) {
```

Comment:
What's the reasoning behind this change?

Reply:
It's no longer necessary, since source files are created first (they used to be created last).